requirements.txt

```diff
@@ -0,0 +1,16 @@
+aiohttp==3.3.2
```

Comment:
Not that I've examined a huge amount of python repos, but I think this is unusual. Usually requirements.txt just has the basic requirements (requirements.base), but with some non-specific version requirements (`>=1.2.3, <2.0.0`, etc)

Assuming you want this file for reproduceability reasons, maybe requirements.lock or something?

Reply:
It is the other way around. `requirements.txt` pins the exact version, and often `setup.py` supplies the range. In this case, `requirements.base`., because I was looking too much how Python did their bot :D (and because I won't have a `setup.py`).